models/obj-schema.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const item = mongoose.Schema({
+    category: {type: String, required: true},
+    product: {type: String, required: true},
+    type: {type: String, uppercase: true, enum: ['School', 'Work', 'Personal']}
+});
+
+module.export = mongoose.model('item', item);
```

Comment:
I think that the name of this should be changed because you are going to want to have a category and product schema
categories I would set up with the following fields:
- name
- description
products I would set up with the following fields:
- name 
- description
- price
- category